create.js

```diff
@@ -52,6 +52,7 @@ var LOCALE_VARS = {
     "uk": "Урок",
     "ja": "日課",
     "zh": "每日课程",
+    "ht": "每日課程"
```

Comment:
@imasaru I am pretty sure this will never be used, since `xxx-hant` will reside in `zh`

Reply:
@VitalikL I may not understand you correctly... The strings are different in Simplified and Traditional—at least kangxi-wise.

Reply:
@VitalikL I may not understand you correctly... The strings are different in Simplified and Traditional—at least kangxi-wise.